react-accordion-component/my-app/src/AccordionComponent.jsx

```diff
@@ -0,0 +1,42 @@
+import './Styles.css';
+import { useState } from 'react';
+
+// parent component w/ topics array as a prop
+export default function AccordionComponent({ topics }) {
+  // initializes useState
+  const [chosenTopic, setChosenTopic] = useState();
+
+  // toggles topic based on index - if open topic is clicked - will close, if a closed topic is clicked - will open
+  const handleClick = (index) => {
+    if (chosenTopic === index) {
+      setChosenTopic(null);
```

Comment:
Personally, I think `null` was a mistake in the language, so I try not to use it. I use `undefined` instead. But it's just a personal choice. I see lots of code written with `null`.

Reply:
Thank you for this! I meant to ask you if there is a best practice during Q&A and forgot!